### PR TITLE
fix: swap large values

### PR DESCRIPTION
**Pull request by dgacevic**
https://bayes.atlassian.net/browse/CXPS-535

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift [hunk 1]
```diff
@@ -92,6 +92,8 @@ class SwapCurrencyView: FEView<SwapCurrencyConfiguration, SwapCurrencyViewModel>
         view.keyboardType = .decimalPad
         view.delegate = self
         view.addTarget(self, action: #selector(cryptoAmountDidChange(_:)), for: .editingChanged)
+        view.adjustsFontSizeToFitWidth = true
+        view.minimumFontSize = 20
```
Comment (kenanmamedoff) [inline brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift:96]: please use scale instead of size

Comment (dgacevic) [reply brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift:96]: scale is not available for UITextField

Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift:96]: `view.contentScaleFactor = 0.5`

Comment (dgacevic) [reply brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift]: Thought you were talking about minimumScaleFactor as is on UILabel. Changed this now.

**Review by kenanmamedoff — APPROVED** [review]

**Pull request closed without merging**